### DISCUSSION TITLE
23.11.13 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#61] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 코드 리뷰대로 코드 수정 (스크럼 이후 머지 예정)
- 내일은 어떤 일을 하나요?
    - 포스트맨 에러 해결 PR

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
    - 플레이스트에 노래 저장
    - 플레이리스트 - 유저 연관관계 코드 수정
- 내일은 어떤 일을 하나요?

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  -  아티스트 좋아요 기능 구현
- 내일은 어떤 일을 하나요?
  - 아티스트 저장 기능 구현
  - 사용자와 코드 결합

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 오늘 작업은 못했어요 ㅠㅠ
* 내일은 어떤 일을 하나요?
  * 정산 로직 및 테스트 수정
